cmd/loom/addressmapper.go

```diff
@@ -128,11 +134,25 @@ func ListMappingCmd() *cobra.Command {
 			if err != nil {
 				return errors.Wrap(err, "static call contract")
 			}
-			out, err := formatJSON(&resp)
-			if err != nil {
-				return errors.Wrap(err, "format JSON response")
+			type maxLength struct {
+				From int
+				To   int
+			}
+			ml := maxLength{From: 50, To: 50}
+			addressList := make(map[string]bool)
+			for _, value := range resp.Mappings {
+				fromAddr := loom.UnmarshalAddressPB(value.From).String()
+				toAddr := loom.UnmarshalAddressPB(value.To).String()
+				if addressList[toAddr] || addressList[fromAddr] {
+					continue
+				}
+				addressList[fromAddr] = true
+				addressList[toAddr] = true
+				fmt.Printf("%-*s | %-*s |\n",
+					ml.From, fromAddr,
+					ml.To, toAddr)
 			}
-			fmt.Println(out)
+			fmt.Printf("%-*s | %-*s |\n", ml.From, "From", ml.To, "To")
```

Comment:
The header should go at the top. Also lets drop the final `|` (from the header & the body) - it looks awkward, so the format string should just be `"%-*s | %-*s\n"`